abb_irb2400_support/launch/load_irb2400_12_155.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
+        <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find abb_irb2400_support)/urdf/irb2400_12_155.xacro'" />
+</launch>
```

Comment:
Indentation (same with `20_155`, but see comment later about perhaps not introducing separate files for that variant).